pennylane/transforms/defer_measurements.py

```diff
@@ -380,13 +384,6 @@ def node(x):
 @defer_measurements.custom_qnode_transform
 def _defer_measurements_qnode(self, qnode, targs, tkwargs):
     """Custom qnode transform for ``defer_measurements``."""
-    if tkwargs.get("device", None):
-        raise ValueError(
-            "Cannot provide a 'device' value directly to the defer_measurements decorator "
-            "when transforming a QNode."
-        )
-
-    tkwargs.setdefault("device", qnode.device)
     return self.default_qnode_transform(qnode, targs, tkwargs)
```

Comment:
We should be able to just delete this now.

Reply:
You mean to delete the `_defer_measurements_qnode` completely?